objax/zoo/resnet_v2.py

```diff
@@ -335,6 +367,91 @@ def __init__(self,
+    shape = objax_layer.gamma.value.shape
+    objax_layer.gamma = objax.TrainVar(jn.array(keras_layer.__dict__['gamma'].numpy()).reshape(shape))
+    objax_layer.beta = objax.TrainVar(jn.array(keras_layer.__dict__['beta'].numpy()).reshape(shape))
+    objax_layer.running_mean = objax.StateVar(jn.array(keras_layer.__dict__['moving_mean'].numpy()).reshape(shape))
+    objax_layer.running_var = objax.StateVar(jn.array(keras_layer.__dict__['moving_variance'].numpy()).reshape(shape))
+
+
+def convert_conv_layer(objax_layer, keras_layer):
+    """Convert variables of convolution layer from objax to keras."""
+    objax_layer.w = objax.TrainVar(jn.array(keras_layer.__dict__['kernel'].numpy()))
+    if 'bias' in keras_layer.__dict__ and keras_layer.__dict__['bias'] is not None:
+        bias_shape = (objax_layer.w.value.shape[-1], 1, 1)
+        objax_layer.b = objax.TrainVar(jn.array(keras_layer.__dict__['bias'].numpy()).reshape(bias_shape))
+
+
+def convert_linear_layer(objax_layer, keras_layer):
+    """Convert variables of linear layer from objax to keras."""
+    objax_layer.w = objax.TrainVar(jn.array(keras_layer.__dict__['kernel'].numpy()))
+    if objax_layer.b:
+        bias_shape = objax_layer.b.value.shape
+        objax_layer.b = objax.TrainVar(jn.array(keras_layer.__dict__['bias'].numpy()).reshape(bias_shape))
+
+
+def convert_resblock(objax_block, keras_block):
+    """Convert variables of residual block from objax to keras.
+
+    Name mapping between blocks:
+        proj_conv   : 0_conv (when exists)
+        conv_0      : 1_conv
+        conv_1      : 2_conv
+        conv_2      : 3_conv
+        norm_0      : preact_bn
+        norm_1      : 1_bn
+        norm_2      : 2_bn
+    """
+    if hasattr(objax_block, 'proj_conv'):
+        convert_conv_layer(objax_block.proj_conv, [layer for layer in keras_block if layer.name.endswith('0_conv')][0])
+    convert_conv_layer(objax_block.conv_0, [layer for layer in keras_block if layer.name.endswith('1_conv')][0])
+    convert_conv_layer(objax_block.conv_1, [layer for layer in keras_block if layer.name.endswith('2_conv')][0])
+    convert_conv_layer(objax_block.conv_2, [layer for layer in keras_block if layer.name.endswith('3_conv')][0])
+    convert_bn_layer(objax_block.norm_0, [layer for layer in keras_block if layer.name.endswith('preact_bn')][0])
+    convert_bn_layer(objax_block.norm_1, [layer for layer in keras_block if layer.name.endswith('1_bn')][0])
+    convert_bn_layer(objax_block.norm_2, [layer for layer in keras_block if layer.name.endswith('2_bn')][0])
+
+
+def convert_model(model_keras, model_objax, num_blocks: list, include_top: bool = True):
+    """Convert keras implementation of ResNetV2 into objax."""
+    convert_conv_layer(model_objax[0], model_keras.get_layer('conv1_conv'))
+    for b, j in enumerate(num_blocks):
+        for i in range(j):
+            convert_resblock(model_objax[b + 3][i], [layer for layer in model_keras.layers if layer.name.startswith(
```

Comment:
Line wrapping looks weird, either extract `[layer for layer in ...]` into separate line (by assigning it to the variable) or change line wrapping, for example:

```python
convert_resblock(model_objax[b + 3][i],
                 [layer for layer in model_keras.layers
                  if layer.name.startswith('conv{}_block{}'.format(b + 2, i + 1))])
```